pkgs/development/libraries/openssl/nix-ssl-cert-file.patch

```diff
@@ -5,8 +5,8 @@ diff -ru -x '*~' openssl-1.0.2j-orig/crypto/x509/by_file.c openssl-1.0.2j/crypto
      switch (cmd) {
      case X509_L_FILE_LOAD:
          if (argl == X509_FILETYPE_DEFAULT) {
--            file = (char *)getenv(X509_get_default_cert_file_env());
-+            file = (char *)getenv("NIX_SSL_CERT_FILE");
+-            file = getenv(X509_get_default_cert_file_env());
++            file = getenv("NIX_SSL_CERT_FILE");
 +            if (!file)
 +                file = (char *)getenv(X509_get_default_cert_file_env());
```

Comment:
Should probably remove the `(char *)` here as well to match the upstream code more closely and because the cast isn't actually safe AFAIK.

Reply:
Oops...